rules/concepts.md

```diff
@@ -48,6 +48,151 @@ In some cases, it is useful to have a rule that is only manually triggered.
+
+<!-- TODO: Update reference when actions are included in the reworked rule docs. -->
+Refer to [Thing Status Action](/docs/configuration/actions.html#thing-status-action) to find how to get the new thing status details or description in the script.
+
+### **System**-based Triggers
+
+System-based triggers are provided as described in the table below:
+
+| Trigger                            | Description                                                                                                                                                                                                                                          |
+|------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| System started                     | `System started` is triggered upon openHAB startup. In openHAB version 2, `System started` is also triggered after the rule file containing the System started trigger is modified, or after item(s) are modified in a .items file.                  |
+| System reached start level <level> | `System reached start level <level>` is triggered when openHAB reaches a specific start level. A list of possible start levels is available below. Please note that only levels 40 and higher are useful as the rule engine needs to be ready first. |
+
+You may wish to use the 'System started' trigger to initialize values at startup if they are not already set.
+You can then execute a rule on the next startup level which depends on the value set by the initialization rule.
+
+In openHAB version 3 the System-based Trigger for startlevel had been added, values depends on the startlevel:
+
+```text
+00 - OSGi framework has been started.
+10 - OSGi application start level has been reached, i.e. bundles are activated.
+20 - Model entities (items, things, links, persist config) have been loaded, both from db as well as files.
+30 - Item states have been restored from persistence service, where applicable.
+40 - Rules are loaded and parsed, both from db as well as dsl and script files.
+50 - Rule engine has executed all "system started" rules and is active.
+70 - User interface is up and running.
+80 - All things have been initialized.
+100 - Startup is fully complete.
+```
+
+Startlevels less than 40 are not available as triggers because the rule engine needs to start up first before it can execute any rules.
+Please keep in mind that the rule engines of automation add-ons like JS Scripting can start up after startlevel 100 has been reached.
+
+## Conditions
+
+With triggers, we have the *When __t__ happens* part completed, so next up is the *if __c__* part.
+This part can limit when a rule can run by adding one or more condition(s).
+
+Conditions as an extra part of the rule are only supported in UI-based rules.
```

Comment:
> Conditions are an 

I'm not certain they are only supported in UI-based rules. I think the syntax may not be there but that doesn't mean it couldn't be supported. So I would hedge here and say something like "... rule that some rule engines support." That way when and if it's added to the text based rules we won't have an inconsistency here when we forget to come back and edit it.